chapter2.html

```diff
@@ -225,7 +225,7 @@ <h2>Networked publics</h2>
         <p>change from bi to unidirectional connects as norm with Twitter. asymmetrical disclosure. reinforces weak ties, lowers barrier to communication. [ellison13]</p> -->
 
     <p>First I reflect on the digital substitutions for the physical body in online social interactions. Then, in comparing and contrasting 'old school' style blogging with contemporary (circa 2013-2017) social networking sites I look deeper into how differences in technological affordances impact peoples' interactions and self-expression.</p>
-    <p>Audiences for identity performance as well as the context in which the performance takes place are critical, but online both of these may be unknown or dynamic, or both. I'll introduce work around imagined audience and context collapse, both of which pioneer our understanding of identity behaviours in networked publics. When audiences and contexts are known, we can examine how people connect with others and form communities; in the final section I look at trust, social reinforcement of identity, and studies of what social media participants choose to disclose or conceal.</p>
+    <p>Audiences for identity performance as well as the context in which the performance takes place are critical, but online both of these may be unknown or dynamic, or both. I'll introduce and work around imagined audience and context collapse, both of which pioneer our understanding of identity behaviours in networked publics. When audiences and contexts are known, we can examine how people connect with others and form communities; in the final section I look at trust, social reinforcement of identity, and studies of what social media participants choose to disclose or conceal.</p>
```

Comment:
no, the work is around imagined audience, and I'm introducing it. Agree 'around' is not the best word though.